src/main/java/uk/gov/companieshouse/company/links/service/ChargesService.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.companieshouse.company.links.service;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.charges.ChargesApi;
+import uk.gov.companieshouse.api.http.ApiKeyHttpClient;
+import uk.gov.companieshouse.api.http.HttpClient;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.company.links.exception.RetryableErrorException;
+import uk.gov.companieshouse.logging.Logger;
+
+@Service
+public class ChargesService extends BaseApiClientService {
+
+    @Value("${api.charges-data-api-key}")
+    private String chargesApiKey;
+
+    @Value("${api.charges-data-api-url}")
+    private String chargesApiUrl;
+
+    /**
+     * Construct a company profile service - used to retrieve a company profile record.
+     * @param logger the CH logger
+     */
+    @Autowired
+    public ChargesService(Logger logger) {
+        super(logger);
+    }
+
+    /**
+     * Retrieve company charges given a company number from charges-data-api.
+     *
+     * @param companyNumber the company's company number
+     * @return an ApiResponse containing the charges data model
+     */
+    public ApiResponse<ChargesApi> getCharges(String contextId, String companyNumber)
+        throws RetryableErrorException {
+        String uri = String.format("/company/%s/charges", companyNumber);
+
+        Map<String, Object> logMap = createLogMap(companyNumber, "GET", uri);
```

Comment:
Need to handle non 200 series responses and throw relavent exceptions etc, maybe part of DSND-861?

Reply:
handleResponse method from StreamResponseProcessor does this added call.